packages/web/src/js/Collapse.ts

```diff
@@ -35,17 +37,17 @@ class Collapse extends BaseComponent {
 
   constructor(element: HTMLElement) {
     super(element);
-    this.target = this.element.dataset.target ? selectorEngine.findOne(`#${this.element.dataset.target}`) : null;
+    this.target = this.element.dataset.target ? SelectorEngine.findOne(`#${this.element.dataset.target}`) : null;
     this.parent = this.element && this.element.parentElement;
     this.meta = {
       id: this.element.dataset.target,
       hideOnCollapse: !!(this.element.dataset.more || this.element.dataset.more === ''),
+      parent: this.target?.dataset.parent,
```

Comment:
We should use better naming. Now there are properties `this.parent` and `this.meta.parent`.  It is not clear now which is used for which purpose.

Reply:
Is `meta.accordionParent` better for our purpose?

Reply:
I do not want to use any accordion naming here. Eg. `this.meta.parent` is ok for `data-parent`. But I will prefer to change it. So I think `this.parent` should be the property with the value of `data-parent` and for `this.element.parentElement` we should use for instance a `toggleParent` or `triggerParent` name. From my perspective the trigger parent is used only for hide-on-close and it is not corresponding with main `data-parent` usage.

Reply:
fixed